<launch>
    <arg name="input_can_frame" default="/received_messages"/>
    <arg name="output_can_frame" default="/sent_messages"/>
    <arg name="output_ultra_sonic_radar" default="/sensing/ultra_sonic_radar/ultra_sonic_radar" />

    <node pkg="ultra_sonic_radar_driver" name="ultra_sonic_radar_driver_node" type="ultra_sonic_radar_driver_node">
        <rosparam command="load" file="$(find ultra_sonic_radar_driver)/config/config.yaml" />
        <remap from="/input/can_frame" to="$(arg input_can_frame)"/>
        <remap from="/output/can_frame" to="$(arg output_can_frame)"/>
        <remap from="/output/ultra_sonic_radar_0" to="$(arg output_ultra_sonic_radar)_0"/>
        <remap from="/output/ultra_sonic_radar_1" to="$(arg output_ultra_sonic_radar)_1"/>
        <remap from="/output/ultra_sonic_radar_2" to="$(arg output_ultra_sonic_radar)_2"/>
        <remap from="/output/ultra_sonic_radar_3" to="$(arg output_ultra_sonic_radar)_3"/>
        <remap from="/output/ultra_sonic_radar_4" to="$(arg output_ultra_sonic_radar)_4"/>
        <remap from="/output/ultra_sonic_radar_5" to="$(arg output_ultra_sonic_radar)_5"/>
        <remap from="/output/ultra_sonic_radar_6" to="$(arg output_ultra_sonic_radar)_6"/>
        <remap from="/output/ultra_sonic_radar_7" to="$(arg output_ultra_sonic_radar)_7"/>
    </node>
</launch>